src/openrct2/scripting/bindings/ride/ScTrackSegment.cpp

```diff
@@ -249,10 +249,7 @@ DukValue ScTrackSegment::getMirrorElement() const
 DukValue ScTrackSegment::getAlternativeElement() const
 {
     const auto ctx = GetContext()->GetScriptEngine().GetContext();
-    const auto& ted = GetTrackElementDescriptor(_type);
-    if (ted.alternativeType == TrackElemType::None)
-        return ToDuk(ctx, nullptr);
-    return ToDuk<int32_t>(ctx, EnumValue(ted.alternativeType));
+    return ToDuk(ctx, nullptr);
```

Comment:
Wouldn't this break older plugins that use this property, even if it's deprecated?